doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -189,6 +189,17 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
 Bluetooth LE
 ============
 
+.. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0
+
+NCSDK-31528: Deadlock on sysworkq with tx_notify in Host
+  If a blocking callback is run on the system workqueue when the HCI buffer pool is full, the TX processor will be blocked.
+  This will prevent TX buffers from being freed, deadlocking the application.
+  An example of a blocking callback is calling the :c:func:`bt_conn_get_tx_power_level` function in a receive callback, which is blocking since it uses the :c:func:`bt_hci_cmd_send_sync` function.
```

Comment:
```suggestion
NCSDK-31528: Deadlock on sysworkq with ``tx_notify`` in Host
  If a blocking callback is run on the system workqueue when the HCI buffer pool is full, the TX processor will be blocked.
  This prevents TX buffers from being freed, deadlocking the application.
  An example of a blocking callback is calling the :c:func:`bt_conn_get_tx_power_level` function in a receive callback, which is blocking since it uses the :c:func:`bt_hci_cmd_send_sync` function.
```